resources/assets/js/app.js

```diff
@@ -1363,19 +1365,14 @@ function initAutocomplete() {
             //Finished appending
 
             //Update stats
-            $('#waste-insert').html( json.stats['ewaste'] );
-            $('#co2-insert').html(  json.stats['co2'] );
-            $('#fixed-insert').html(  json.stats['fixed_devices'] );
-            $('#repair-insert').html(  json.stats['repairable_devices'] );
-            $('#dead-insert').html(  json.stats['dead_devices'] );
-
-            //Give users some visual feedback
```

Comment:
Presume this visual feedback on the button no longer make sense?

Reply:
Yes, I think so.  I've tried to ensure that there is some visual feedback.  

What I have for Freegle is a Vue component called SpinButton.  Buttons typically have icon + text, and when you click the icon turns into a rotating refresh sort of thing while the action is happening.  That means you have consistency across the site.  At the moment it looks like here we have code that's specific to individual cases.

Reply:
Yes, from what I've seen it's specific to individual cases at present.